Validation/Geometry/test/genTrackerPlots.sh

```diff
@@ -9,7 +9,13 @@ mkdir $TEST_DIR && cd $TEST_DIR
 cmsRun ${VGEO_DIR}/test/single_neutrino_cfg.py nEvents=1000 >$TEST_DIR/single_neutrino_cfg.log 2>&1
 
 for geom in {'Extended2015','Extended2017Plan1'}; do
-    python ${VGEO_DIR}/test/runP_Tracker.py geom=$geom label=Tracker >$TEST_DIR/runP_Tracker_cfg.log 2>&1
+  {   
+   python3 -c "from FWCore.PythonFramework.CmsRun import CmsRun" 2>/dev/null && python3 ${VGEO_DIR}/test/runP_Tracker.py geom=$geom label=Tracker >$TEST_DIR/runP_Tracker_cfg.log 2>&1
+  }|| 
+  { 
+  python -c "from FWCore.PythonFramework.CmsRun import CmsRun" 2>/dev/null && python2 ${VGEO_DIR}/test/runP_Tracker.py geom=$geom label=Tracker >$TEST_DIR/runP_Tracker_cfg.log 2>&1
+  } 
+    
```

Comment:
@davidlange6,  here I would suggest to add something like the following before the `for` loop and then use `$python_cmd ....` in the loop and after the loop too
```
python_cmd="python2"
python3 -c "from FWCore.PythonFramework.CmsRun import CmsRun" 2>/dev/null && python_cmd="python3"
```

better to use this logic in all 3 scripts :-)

Reply:
ah, that is better! - I'll get that changed when I can (maybe not immediately). Thanks!

Reply:
@davidlange6 , I updated scripts to use the 
```
python_cmd="python2"
python3 -c "from FWCore.PythonFramework.CmsRun import CmsRun" 2>/dev/null && python_cmd="python3"
```

Reply:
Thanks @smuzaffar !